Bricks/StringUtils/0.1.0.toml

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
Note: You need to update this file name to `0.1.1.toml`

Reply:
@ben-albrecht  carelessness from my side, sorry for that.

Reply:
I think it's fine now. Please let me know.